src/cmd/prepare.go

```diff
@@ -90,7 +90,7 @@ var prepareComputeFileSha256sum = &cobra.Command{
 var prepareFindImages = &cobra.Command{
 	Use:     "find-images [ PACKAGE ]",
 	Aliases: []string{"f"},
-	Args:    cobra.MaximumNArgs(1),
+	Args:    cobra.ExactArgs(1),
```

Comment:
```suggestion
	Args:    cobra.MaximumNArgs(1),
```

`zarf prepare find-images` is a thing (no args)